metakb/harvesters/pmkb.py

```diff
@@ -0,0 +1,245 @@
+"""PMKB harvester"""
+import logging
+from .base import Harvester
+from metakb import PROJECT_ROOT, FileDownloadException
+import requests
+import re
+from typing import List
+import pandas as pd
+import numpy as np
+import json
+from pathlib import Path
+
+
+logger = logging.getLogger('Harvesters')
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler())
+
+
+class PMKB(Harvester):
+    """Harvester class for Weill Cornell PMKB"""
+
+    def __init__(self):
+        """Set up harvester object"""
+        self.assertions = []
+
+    def harvest(self, data_dir: Path = PROJECT_ROOT / 'data' / 'pmkb') -> bool:
+        """Harvest PMKB source. Retrieve and store genes, variants, and
+        interpretations.
+
+        :param pathlib.Path data_dir: path to local PMKB data source directory
+        :return: `True` if successful, `False` otherwise.
```

Comment:
Does this ever return False?

Reply:
Good catch, I meant to return to this -- IMO down the line we might want to rethink whether returning true/false from this method is an optimal way of managing success and failure (not saying it's not, just seems like it's worth talking out). I'll add a basic exception catch in the meantime -- but I think at that point we could also go back and make some rules for how we want to catch generic exceptions (there are apparently small differences between a catch-all `except`, `except Exception`, and `except BaseException`, see eg https://realpython.com/the-most-diabolical-python-antipattern/)